File: 2-Basic-Mathematical-Knowledge-for-Algorithms/002-Sum_of_3_Integers/go/main.go
package main

import "fmt"

func main() {
	// 1. 入力処理
	var a1, a2, a3 int
	fmt.Scan(&a1, &a2, &a3)

	// 2. 計算処理
	sum := a1 + a2 + a3

	// 3. 出力処理
	fmt.Println(sum)
}

File: 2-Basic-Mathematical-Knowledge-for-Algorithms/002-Sum_of_3_Integers/java/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 1. 入力処理
        Scanner sc = new Scanner(System.in);
        int a1 = sc.nextInt();
        int a2 = sc.nextInt();
        int a3 = sc.nextInt();
        sc.close();

        // 2. 計算処理
        int sum = a1 + a2 + a3;

        // 3. 出力処理
        System.out.println(sum);
    }
}